modules/ROOT/pages/database-internals/locks-deadlocks.adoc

```diff
@@ -216,14 +216,15 @@ The locking is very similar for sparse and dense nodes.
 The biggest contention for sparse nodes is the update of the degree (i.e. number of relationships) for the node.
 Dense nodes store this data in a concurrent data structure, and so can avoid exclusive node locks in almost all cases for relationship modifications.
 
+[[transaction-management-lock-acquisition-timeout]]
+=== Configure lock acquisition timeout
+
 [NOTE]
 ====
-This section is only relevant for `standard`, `aligned`, and `high-limit` formats.
+This section is only applies to `standard`, `aligned`, and `high-limit` store formats. 
+It does not apply to `block` format. 
```

Comment:
This change is wrong. The note belongs to the `Locks for dense nodes` section.
Most of it is true for block format also, but the settings and some details are not

Reply:
Should I move this note below the heading `=== Locks for dense nodes`? I am not sure what to do with it?